backend/apps/owasp/score_calculator.py

```
@@ -0,0 +1,336 @@
+
+        """
+        # Get all indexed users with contributions
+        users_with_contributions = User.objects.filter(
+            created_pull_requests__isnull=False,
+        ).distinct()
+
+        total_users = users_with_contributions.count()
+        updated_count = 0
+        created_count = 0
+
+        logger.info(f"Starting score recalculation for {total_users} users")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)